user-account-management.md

```diff
@@ -65,69 +59,49 @@ The `test` user is allowed to log in from any hosts on the `192.168.10` subnet.
 
 If the host is not specified, the user is allowed to log in from any IP. If no password is specified, the default is empty password:
 
-{{< copyable "sql" >}}
```

Comment:
Why remove copyable?
The effect of this is that the use can copy this sql and try it themselves

Reply:
@tiancaiamao This copy syntax is no longer needed for the current docs website, as all code blocks are now copyable by default.